tests/test_url.py

```diff
@@ -131,13 +131,27 @@ def test_norm_quote (self):
         url = "http://example.com/?u=http://example2.com?b="
         nurl = "http://example.com/?u=http://example2.com?b="
         self.urlnormtest(url, nurl)
-        url = "http://localhost:8001/?quoted=�"
```

Comment:
I hate this kind of thing in a test file where the contents of a variable depend on the encoding of the file.

The `# -*- coding: iso-8859-1 -*-` at the top implies this was "...quoted=\xfc".